Project_source/app/src/main/java/android/clicker/school_live_simulator/MacbookState.kt

```diff
@@ -0,0 +1,13 @@
+package android.clicker.school_live_simulator
+
+open class MacbookState: XiaomiMiNotebookState() {
```

Comment:
You have to add one more space here in every child class
```suggestion
open class MacbookState : XiaomiMiNotebookState() {
```

Reply:
Fixed